### PR TITLE
Added Installing Script for Ubuntu

### PR DESCRIPTION
**Pull request by anshuman73**
Script to install MongoDB on Ubuntu and start the service

### CODE REVIEW

File: ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# mount the local-ssd
+mkfs.ext3 /dev/nvme0n1
+mount /dev/nvme0n1 /mnt
```
Comment (sudheesh001) [inline ubuntu.sh]: Do we really need a mount step in the installation script ? I'll leave the decision to @mnmtanish

Comment (anshuman73) [inline ubuntu.sh]: His original script contained this. I believe he wanted the MongoDB to be installed on a mount.
https://github.com/rmdb/benchmark-mongodb/blob/v3.2/scripts/setup.sh

Comment (mnmtanish) [inline ubuntu.sh]: @sudheesh001
I'm running these benchmarks on Google Compute Engine these days so it's cool.

@anshuman73
I'm creating a new pull-request to the **v3.2** branch https://github.com/rmdb/benchmark-mongodb/pull/4

Comment (anshuman73) [inline ubuntu.sh]: sure !

**Pull request closed without merging**